Instances.py

```diff
@@ -0,0 +1,22 @@
+# import the Particle class from the particle module
+import particle as p
+
+# create an empty list to hold observed particle data
+obs = []
+
+# append the first particle; by the way, append means to add
+obs.append(p.Particle())
```

Comment:
running this code fails due to the missing argument in Particle(). This method is currently defined in particle.py as requiring 4 arguments; self, charge, mass, position. So the line should look like, 

obs.append(p.Particle(1,1,1))

Making this change allowed for the code to run correctly.